sos/__init__.py

```diff
@@ -191,5 +191,6 @@ def _init_component(self):
 
     def execute(self):
         self._component.execute()
+        os._exit(0)
```

Comment:
Ah, this would apparently actually need to go in `bin/sos` after `sos.execute()` is called there. We use this same hook when calling cleaner in-line from report, so we're prematurely exiting in that case.